packages/volto/razzle.config.js

```diff
@@ -356,6 +356,15 @@ const defaultModify = ({
     };
   }
 
+  const prefixPath = process.env.RAZZLE_PREFIX_PATH || '';
+
+  if (prefixPath) {
+    if (target === 'web' && dev) {
+      config.devServer.devMiddleware.publicPath = prefixPath;
+    }
+    const pp = config.output.publicPath;
```

Comment:
No cryptic vars, please.

And regarding the dev server, is it necessary to also use the prefix while developing?

Reply:
In webpack 5, the public path in dev mode needs to be prefixed as well. `http://localhost:3001/foo/static/js/client.js`